packages/rlc-common/src/helpers/apiVersionUtil.ts

```diff
@@ -0,0 +1,54 @@
+import { ApiVersionInfo, ApiVersionPosition, UrlInfo } from "../interfaces";
+
+/**
+ * Extract the path api-version detail from UrlInfo, return undefined if no valid api-version parameter
+ * @param urlInfo UrlInfo detail
+ * @returns path api-version detail
+ */
+export function extractPathApiVersion(
+  urlInfo?: UrlInfo
+): ApiVersionInfo | undefined {
+  if (!urlInfo) {
+    return;
+  }
+  const param = urlInfo.urlParameters?.filter(
+    (p) =>
+      p.name.toLowerCase() === "api-version" ||
+      p.name.toLowerCase() === "apiversion"
+  );
+  if (!param || param?.length < 1) {
+    return;
+  }
+  const detail: ApiVersionInfo = {
+    definedPosition: "query",
```

Comment:
should the definedPosition for this scenario be `path` ?

Reply:
Good catch!